### PR TITLE
T9997: Add Miraheze-specific CreateWiki messages

### PR DESCRIPTION
**Pull request by redbluegreenhat**
This pull request adds some Miraheze-specific CreateWiki messages as overrides.

These overrides were copied from https://github.com/miraheze/CreateWiki/blob/4c536439ec110888f2a148d3ced2c550f1563300/i18n/en.json, with requestwiki-header being from https://github.com/miraheze/CreateWiki/pull/348/commits/155be0dad89af317fc1515cee641de1c04aa0b4c.

Please see https://phabricator.miraheze.org/T9997.

### CODE REVIEW

File: i18n/overrides/en.json [hunk 1]
```diff
@@ -45,5 +45,14 @@
...
...as needed. To start, try checking out these helpful links:\n* [[mw:Special:MyLanguage/Help:Contents|MediaWiki guide]] (e.g. navigation, editing, deleting pages, blocking users)\n* [[meta:Special:MyLanguage/FAQ|Miraheze FAQ]]\n* [[meta:Special:MyLanguage/Request features|Request settings changes on your wiki]]. (Extensions, Skin and Logo/Favicon changes should be done through [[Special:ManageWiki]] on your wiki, see [[meta:Special:MyLanguage/ManageWiki|ManageWiki]] for more information.)\n\n==== I still don't understand X! ====\nWell, that's no problem. Even if something isn't explained in the documentation/FAQ, we are still happy to help you. You can find us here:\n* [[meta:Special:MyLanguage/Help center|On our own Miraheze wiki]]\n* On [[phab:|Phabricator]]\n* On [https://miraheze.org/discord Discord]\n* On IRC in #miraheze on irc.libera.chat ([irc://irc.libera.chat/%23miraheze direct link]; [https://web.libera.chat/?channel=#miraheze webchat])\n\n=== For visitors of this wiki ===\nHello, the default Main Page of this wiki (this page) has not yet been replaced by the bureaucrat(s) of this wiki. The bureaucrat(s) might still be working on a Main Page, so please check again later!",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:49]: All these messages need to be prefixed with `miraheze-` for overrides.

File: i18n/overrides/en.json [hunk 2]
```diff
@@ -45,5 +45,14 @@
...
+	"requestwiki-error-invalidcomment": "Your wiki request contains an invalid character. Please try requesting again, and if the problem persists [[Help center|please contact us]]",
+	"requestwiki-error-patient": "You have already requested a wiki. Please be patient and one of our wiki creators should create it soon. If you need to request another wiki, please wait and you will be able to request later.",
+	"requestwiki-header": "Thank you for choosing Miraheze to host your wiki! To get started, please fill out this form so we can review your request.\nYour wiki request will be reviewed by a volunteer, and it can be tracked at the page that will be given when submitting your request.\n\nIf you want a custom domain for your wiki (i.e. not a standard .miraheze.org subdomain), please read <span class=\"plainlinks\">[$1 this]</span> first. You must first choose a subdomain, and your custom domain will be configured afterwards. \n\nWhen you request a wiki, you automatically agree to our [[Content Policy|Content]] and [[Dormancy Policy]].",
+	"requestwiki-label-siteurl": "Subdomain: (subdomain.miraheze.org)",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:56]: This one in particular I think can be left in CreateWiki. I intend to make the domain example be a `$1` value later, that is based on domain configuration.

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
Some things here. Additionally, maybe some of miraheze/CreateWiki#348 could be integrated here to sync Miraheze messages with Meta?

File: i18n/overrides/en.json (hunk 2 above)
Comment (redbluegreenhat) [reply i18n/overrides/en.json]: Done

File: i18n/overrides/en.json (hunk 1 above)
Comment (redbluegreenhat) [reply i18n/overrides/en.json]: Done.

**Comment by redbluegreenhat** [comment]
> Some things here. Additionally, maybe some of [miraheze/CreateWiki#348](https://github.com/miraheze/CreateWiki/pull/348) could be integrated here to sync Miraheze messages with Meta?
What do you want from that PR? Just the messages or also the "ActionFilteredLogs" change?

**Comment by redbluegreenhat** [comment]
Decided to add only the messages, as it doesn't make much sense to add the ActionFilteredLogs change here, my bad.

File: i18n/overrides/en.json [hunk 3]
```diff
@@ -45,5 +46,20 @@
...
+	"miraheze-requestwiki-success": "Your wiki request ($1) was successfully submitted. Your request will be reviewed by a volunteer wiki creator as soon as possible.",
+	"miraheze-createwiki-error-disallowed": "The subdomain you have requested cannot be created. Please either request a new subdomain or contact system administrators for help.",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:58]:
```suggestion
	"miraheze-createwiki-error-disallowed": "The subdomain you have requested cannot be created. Please either request a new subdomain or contact Site Reliability Engineers for help.",
```

File: i18n/overrides/en.json [hunk 4]
```diff
@@ -45,5 +46,20 @@
...
+	"miraheze-log-action-filter-farmer-requestaccept": "Wiki request approved",
+	"miraheze-log-action-filter-farmer-requestwiki": "Wiki request (re)opened",
+	"miraheze-logentry-farmer-requestaccept": "$1 approved wiki request $4",
+	"miraheze-logentry-farmer-requestdecline": "$1 declined wiki request $4",
+	"miraheze-logentry-farmer-requestonhold": "$1 held wiki request $4",
+
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:64]:
```suggestion
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
Just some very minor things here.

File: i18n/overrides/en.json [hunk 5]
```diff
@@ -45,5 +46,20 @@
...
+	"miraheze-requestwiki-success": "Your wiki request ($1) was successfully submitted. Your request will be reviewed by a volunteer wiki creator as soon as possible.",
+	"miraheze-createwiki-error-disallowed": "The subdomain you have requested cannot be created. Please either request a new subdomain or contact system administrators for help.",
+	"miraheze-log-action-filter-farmer-requestaccept": "Wiki request approved",
+	"miraheze-log-action-filter-farmer-requestwiki": "Wiki request (re)opened",
+	"miraheze-logentry-farmer-requestaccept": "$1 approved wiki request $4",
+	"miraheze-logentry-farmer-requestdecline": "$1 declined wiki request $4",
+	"miraheze-logentry-farmer-requestonhold": "$1 held wiki request $4",
```
Comment (Universal-Omega) [inline i18n/overrides/en.json:63]:
```suggestion
	"miraheze-logentry-farmer-requestonhold": "$1 held wiki request $4"
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]
Oh, one more thing, a syntax error.

**Comment by Universal-Omega** [comment]
log-action-filter messages are pretty generic also, might be able to stay in CreateWiki changed like that also.

**Comment by redbluegreenhat** [comment]
> log-action-filter messages are pretty generic also, might be able to stay in CreateWiki changed like that also.

Done.

File: includes/MirahezeMagicHooks.php [hunk 6]
```diff
@@ -332,6 +332,16 @@ public static function onMessageCacheGet( &$lcKey ) {
...
 			'group-bot-member',
 			'grouppage-user',
+			'createwiki-defaultmainpage',
+			'createwiki-close-email-sender',
+			'createwiki-email-subject',
+			'createwiki-email-body',
+			'requestwiki-error-invalidcomment',
+			'requestwiki-error-patient',
+			'requestwiki-header',
+			'requestwiki-label-siteurl',
```
Comment (Universal-Omega) [inline includes/MirahezeMagicHooks.php:342]: This needs removed since no longer overriding.
```suggestion
```

**Review by Universal-Omega — CHANGES_REQUESTED** [review]

**Comment by redbluegreenhat** [comment]
main page message moved.

**Pull request closed without merging**